src/index.tsx

```diff
@@ -1,10 +1,18 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
+import { store } from "./store/store";
+
+//REDUX
+// import { Provider } from "react-redux";
+// import store from "./store/store";
 
 ReactDOM.render(
+  // <Provider store={store}>
```

Comment:
Tutaj chcemy to odkomentować na pewno i chyba musisz dodać reducer jeszcze bo nigdzie nie widziałem